modules/specialized-guides/pages/salt/salt-monitoring.adoc

```diff
@@ -0,0 +1,36 @@
+[[salt-monitoring]]
+
+= Monitoring salt events
+
+In some cases {salt} event bus could publish significant amount of events with the high rate, what could make it hard to analize.
+Getting the statistics based on the events published in the {salt} event bus can help to identify possible weak points and perform {salt} internals profiling.
+
+[[saline]]
+= Saline
+
+Saline is a tool which can be used as Prometheus exporter to get the metrics based on the events from the {salt} event bus.
+Saline is getting the statistics from the events which can also be retrived from the {salt} master with [command]``mgrctl exec -ti -- salt-run state.event``.
+Besides of the Prometheus exporter Saline also provides [literal]``saline-formula``, which can be used to configure Prometheus and Grafana dashboards.
+
+[[saline-deployment]]
+== Saline deployment
+
+To deploy Saline on the {productname} Server use [command]``mgradm upgrade podman`` with [option]``--saline-replicas 1``.
+Saline can be deployed during the installation of the {productname} Server using [command]``mgradm install podman`` with [option]``--saline-replicas 1``.
+The only possible values for [option]``--saline-replicas`` are ``1`` or ``0``.
+The default port [option]``8216`` can be changed with specifying [option]``--saline-port NNNN`` (where [literal]``NNNN`` is a desired port number) on calling [command]``mgradm upgrade podman``.
+After finishing the deployment [literal]``uyuni-server-saline@0.service`` should be configured on the {productname} Server host system.
+
+== {salt} master configuration recommendations
+
+It's recommended to add [option]``master_stats: True`` and adjust the value of [option]``master_stats_event_iter: 60`` if needed and restart [command]``salt-master``,
+to get the detailed statistics of internal salt calls, what can be very helpful for tweaking large scale deployment installations.
```

Comment:
```suggestion
It is recommended to add [option]``master_stats: True`` and adjust the value of [option]``master_stats_event_iter: 60`` if needed.
Then restart [command]``salt-master`` to get the detailed statistics of internal {salt} calls.
This can be help with tuning large scale deployment installations.
```